File: 1929.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int length = nums.length;

        if(length == 0) {
            return nums;
        }

        int[] answer = new int[2*length];

        // O(n) TC
        for(int i=0;i<length;i++) {
            answer[i] = nums[i];
            answer[i+length] = nums[i];
        }
        return answer;
    }
}

File: 1929.py
from typing import List

class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        length = len(nums)
        answer = [0]*(2*length)

        if length == 0:
            return nums
        for i in range(length):
            answer[i] = nums[i]
            answer[i+length] = nums[i]
        return answer